### PR TITLE
Add Vue Browse News Component

### PR DESCRIPTION
**Pull request by probablybenallen**

### CODE REVIEW

File: News/index.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+</div>
+
+{embed="global/.footer" vue_comp="vue-news"}
+
+<script src="https://cdn.jsdelivr.net/npm/select2@4.1.0-rc.0/dist/js/select2.min.js"></script>
+<!--
+<script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/axios/dist/axios.min.js"></script>
+<script src="/test-environment/vue-news.js"></script>
+-->
```
Comment (nathanbillis) [inline News/index.html]: Just a nitpick here to remove the commented out scripts

**Comment by nathanbillis** [comment]
Also need to update the main readme!

**Review by nathanbillis — CHANGES_REQUESTED** [review]

File: News/index.html (hunk 1 above)
Comment (probablybenallen) [reply News/index.html]: Whoops 👀 fixed!!

**Comment by probablybenallen** [comment]
> Also need to update the main readme!

Resolved by 6b0d713ae2abbd87ab38716e9358420e2bcbeb19

**Pull request merged**